Packs/IntSight/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IntSights
+- ***Breaking Change:*** The *intsights-get-iocs* and *intsights-get-ioc-by-value* commands return different outputs according to the new api endpoints.
```

Comment:
Let's specify the new API/product version
```suggestion
- ***Breaking Change:*** The *intsights-get-iocs* and *intsights-get-ioc-by-value* commands return different outputs according to the new API endpoints.
```

Reply:
Intsights broke the api, which is where this bug came from. No reason to keep BC for the apis